discord-bot/src/utils/slack.ts

```diff
@@ -0,0 +1,61 @@
+import { slackUtils, slackBuilder, TBlocks } from 'slack-utility'
+import { config, log } from './config'
+
+export const sendSlackMessage = async (message: string, blocks: TBlocks, messageIdToEdit?: string) => {
+  if (config.SLACK_ENABLED) {
+    try {
+      if (messageIdToEdit) {
+        const slackMsg = await slackUtils.slackUpdateMessage(
+          config.SLACK_TOKEN,
+          config.SLACK_CONVERSATION_ID,
+          message,
+          messageIdToEdit,
+          blocks,
+          false,
+          false,
+          false,
+        )
+        return slackMsg.ts ?? undefined
+      } else {
+        const slackMsg = await slackUtils.slackPostMessage(
+          config.SLACK_TOKEN,
+          config.SLACK_CONVERSATION_ID,
+          message,
+          blocks,
+          false,
+          false,
+          false,
+        )
+        return slackMsg.resultPostMessage.ts ?? undefined
+      }
+    } catch (e) {
+      log('Error sending slack message', e)
+    }
+  }
+}
+
+export const buildSlackStatsMessage = (type: 'weekRecap' | 'update', requestCount: number): TBlocks => {
+  const blocks = [slackBuilder.buildSimpleSlackHeaderMsg('Faucet stats')]
+  switch (type) {
+    case 'weekRecap':
+      blocks.push(slackBuilder.buildSimpleSectionMsg(`Last week total requests: ${requestCount} :subspace-hype:`))
```

Comment:
Let's also include the information about the number of unique addresses that initiated the requests to get an understanding of how many different people are creating the requests. WDYT?

Reply:
Good idea :)